zipline/assets/_assets.pyx

```diff
@@ -299,6 +314,75 @@ cdef class Future(Asset):
         super_dict['contract_multiplier'] = self.contract_multiplier
         return super_dict
 
+"""
+TODO MB - rework this we need a Currency object and we need a FX pair
+
+ok we should use this a CurrencyPair, Currency could just be an enum?
+"""
+
+cdef class CurrencyPair(Asset):
+    cdef readonly object pair # 'USDGBP' for example
+    cdef readonly object base # 'USD'
+    cdef readonly object quote # 'GBP'
```

Comment:
Is this redundant with the 'currency' or 'quote_currency' field of the Asset class?